test/unit/Dredd-test.js

```diff
@@ -588,112 +588,14 @@ GET /url
         },
       };
       dredd = new Dredd(configuration);
-      sinon.stub(dredd.runner, 'executeTransaction').callsFake((transaction, hooks, callback) => callback());
+      sinon.stub(dredd.transactionRunner, 'executeTransaction').callsFake((transaction, hooks, callback) => callback());
     });
 
-    afterEach(() => dredd.runner.executeTransaction.resetHistory());
+    afterEach(() => dredd.transactionRunner.executeTransaction.resetHistory());
 
     it('should execute the runtime', done => dredd.run(() => {
-      assert.isOk(dredd.runner.executeTransaction.called);
+      assert.isOk(dredd.transactionRunner.executeTransaction.called);
       done();
     }));
   });
-
-  describe('#emitStart', () => {
```

Comment:
I removed these unit tests now as they're specific to Dredd.js and would be hard to move to the transaction runner tests. In fact I have integration tests replacing most of this file, covering the basic use cases, waiting in the https://github.com/apiaryio/dredd/pull/1289